types/office-js-preview/index.d.ts

```diff
@@ -84156,6 +84156,56 @@ declare namespace Word {
          * @param propertyNamesAndPaths `propertyNamesAndPaths.select` is a comma-delimited string that specifies the properties to load, and `propertyNamesAndPaths.expand` is a comma-delimited string that specifies the navigation properties to load.
          */
         load(propertyNamesAndPaths?: OfficeExtension.LoadOption): Word.CommentCollection;
+        /**
+         * Occurs when new comments are added.
```

Comment:
Should this be "comment or reply"?

Reply:
If a reply is added, we will trigger the onCommentChanged event.